addon/components/o-s-s/input-container.ts

```diff
@@ -1,9 +1,13 @@
 import { action } from '@ember/object';
 import Component from '@glimmer/component';
 
+type MessageType = 'error' | 'alert' | 'success';
+
 interface OSSInputContainerArgs {
   value?: string;
   disabled?: boolean;
+  successMessage?: string;
+  alertMessage?: string;
   errorMessage?: string;
```

Comment:
This works but I wonder if we would not be better off with a single param for the "message" + a param for the type (error/alert/success)

I'm thinking that if we keep with this current implementation, we will need to defined multiple tracked variables in the parent components if we ever need to handle mutliple types. Versus having a single "message" tracked var + changing the type of the messageType variable.

This is a question for everyone ^^

Reply:
If we do move on to what I am proposing, we can/should keep the existing `errorMessage` working for backwards compatibility until we do a full cleanup in the platform. (This cleanup could be a small team effort during the next cooldown for instance)

Reply:
Good point, this would also solve the question of "what should happen when we set multiple messages at once"

Reply:
+1 on this. I think it leaves the door open for more configuration on how we display messages in the future.

Also @Miexil, maybe we should have analysis cross-squad validated too?  never thought about it but it would make sense for you to see what we plan to add in x-squad repos before we actually implement it, no? 🤔

Reply:
Indeed ahah ^^